src/styles/platform-navigator.scss

```diff
@@ -6,6 +6,8 @@
   &__button {
     display: flex;
     justify-content: space-between;
+    white-space: nowrap;
```

Comment:
React Native title was wrapping

Reply:
I'm wondering if we should just decrease the font size or maybe make the arrow smaller? It still looks really cramped:

![image](https://github.com/aws-amplify/docs/assets/54393192/087c9024-80e3-4f5d-8592-70fc0e9eeee3)